packages/server/src/database/migrations/20200326212213-create-keeper-suspicious-transactions.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) =>
+    queryInterface.createTable('KeeperSuspiciousTransactions',
+      {
+        id: {
+          allowNull: false,
+          autoIncrement: true,
+          primaryKey: true,
+          type: Sequelize.INTEGER
+        },
```

Comment:
I wonder if we should make this table be based on the keeper address—as it is right now, there's only one row anyway, right?

Reply:
Do you mean storing entries per keeper address to support many of them?

Reply:
Yes, I think that's what I meant before, in case we had multiple addresses or swapped the current one out.

Reply:
Cool! I'll create an issue for that